webui/src/pages/bug/Bug.tsx

```diff
@@ -14,9 +14,9 @@ import LabelMenu from './labels/LabelMenu';
  */
 const useStyles = makeStyles((theme) => ({
   main: {
-    maxWidth: 1000,
+    maxWidth: 1100,
```

Comment:
do we want to make this even larger, like around 1400-1600? i found that to be pretty comfortable on displays with widths between 2256 and 2560.

Reply:
I want to rework the layout to be flex-based, and then we can use the breakpoints to make this whatever we'd like; an `xxl` breakpoint that maxes out at 1600 makes sense to me.

Reply:
but what about my buddy who has that ridiculous 5120x1440 samsung display? it'll look like a bug tracker for ants!

(i jest, i think 1600 is plenty fine for a max width)

Reply:
refactoring to be flex-based sounds good to me. we can hold off on dimension-based refactoring until then.